docs/cloud.rst

```diff
@@ -502,3 +289,77 @@ to the Dask scheduler and workers. For example,
    ... }
    >>> options.environment = env
    >>> cluster = gateway.new_cluster(options)
+   
+Dask Software Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default image used on Dask Clusters (the scheduler and workers) matches
+the image used for JupyterHub. It won't, however, have changes you've made
+in your "local" environment in your home directory on the hub.
+
+Long-term, the best way to add packages to the environemnt is by updating the
+Docker images, as described in :ref:`cloud.software_environment`. But for quickly
+prototyping something on the Dask cluster you can use a
+`Dask WorkerPlugin <https://distributed.dask.org/en/latest/plugins.html#distributed.diagnostics.plugin.WorkerPlugin>`_.
+This lets you inject a bit of code that's run when the worker starts up. The `PipPlugin` below
+uses pip to install a configurable list of packages.
+
+
+.. code-block:: python
+
+   import subprocess
+   import logging
+   from distributed import WorkerPlugin
+
```

Comment:
This exists in distirbuted now: https://distributed.dask.org/en/stable/plugins.html#distributed.diagnostics.plugin.PipInstall